etc/spack/defaults/packages.yaml

```diff
@@ -25,9 +25,13 @@ packages:
       fftw-api: [fftw, amdfftw]
       flame: [libflame, amdlibflame]
       fuse: [libfuse]
-      gl: [mesa+opengl, mesa18, opengl]
       glu: [mesa-glu, openglu]
-      glx: [mesa+glx, mesa18+glx, opengl]
+      gl: [libglvnd-fe, mesa~glvnd, mesa18, opengl~glvnd]
+      glx: [libglvnd-fe+glx, mesa+glx~glvnd, mesa18+glx, opengl+glx~glvnd]
+      egl: [libglvnd-fe+egl, opengl+egl~glvnd]
+      libglvnd-be-gl: [mesa+glvnd, opengl+glvnd]
```

Comment:
If I'm understanding this correctly, shouldn't this be `libglvnd-be-gl: [mesa+opengl+glvnd, opengl+glvnd]`?

Reply:
I thought we wanted to remove `+opengl` from the mesa versions of `libglvnd-be` pacakges. Or should those be only removed from mesa versions in `libglvnd-be-glx` and `libglvnd-be-egl`?

Reply:
As discussed in our call, it looks like we can drop the variants from all the provider entries and just list the packages, letting the concretizer enable the necessary variants as appropriate